docs/core/retrieval-actions.rst

```diff
@@ -234,4 +238,8 @@ Example result:
     }
 
 If the ``retrieval_intent`` parameter of a particular response selector was left to its default value,
-the corresponding response selector will be identified as ``default`` in the returned output.
\ No newline at end of file
+the corresponding response selector will be identified as ``default`` in the returned output.
+
+.. warning::
+
+   Since the feature is experimental, retrieval actions are not supported by Rasa-X and interactive learning within Rasa. Using them may lead to unexpected behaviour.
```

Comment:
creating them is not supporting, but using them should work? @amn41 @tmbo

Reply:
@Ghostvv No, I don't think so. Even prediction of retrieval actions while talking to the bot does not work as pointed out by this user - https://forum.rasa.com/t/supervised-response-selector-experimental/13714/2?u=dakshvar22